templates/common-templates.adoc

```diff
@@ -98,13 +98,13 @@ https://docs.openshift.com/enterprise/3.0/dev_guide/templates.html#creating-from
 the templates produce VM objects to be used in the cluster. The VMs
 produced from templates will have a `vm.cnv.io/template` label, whose
 value will be the name of the parent template, for example
-`fedora-generic-medium`:
+`fedora-server-medium`:
```

Comment:
Didn't we change fedora-generic to fedora-desktop?

Reply:
Fedora has all three workloads, so it depends what we choose.

Reply:
Ack, please use fedora-desktop here to have some diversity